src/pharmpy/statements.py

```diff
@@ -593,6 +590,38 @@ def set_lag_time(self, compartment, lag_time):
         return new_comp
 
 
+def _is_positive(expr: sympy.Expr) -> bool:
```

Comment:
It assumes every symbol to be positive which might not be the case. What if there is a covariate in the system or a theta?

Reply:
Yep. We will need a global view of the model if we want to be able to do anything better. However, this change still has value as it fixes an existing issue with the current more naive implementation. Discussing how to improve this even further should not be part of this PR.

Reply:
Agree